modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionsOverview/components/AccountSubscriptionsList/components/AccountSubscriptionCard/AccountSubscriptionCard.jsx

```diff
@@ -45,6 +51,56 @@ const AccountSubscriptionCard = ({
 			FORMAT_DATE_TYPES.day2DMonth2DYearN
 		)}`;
 
+	const {data: accountSubscriptionUsageData} = useGetAccountSubscriptionUsage(
+		accountSubscription?.accountKey,
+		accountSubscription?.productKey
+	);
+
+	const currentConsumption =
+		accountSubscriptionUsageData?.getAccountSubscriptionUsage
+			?.currentConsumption;
+
+	const DisplayonCard = {
```

Comment:
Please, change this object name.
```suggestion
	const DisplayOnCard = {
```

Reply:
Thx Silas :no_good_man: